nxcpy/components/_connected.pyx

```diff
@@ -0,0 +1,32 @@
+from libc.stdlib import malloc, free
+cimport _api.connected
+
+import networkx as nx
+
+__all__ = [
+        'connected_components',
+        'is_connected'
+]
+
+cdef enum Color:
+    WHITE = 0
+    GRAY = 1
+    BLACK = 2
+
```

Comment:
Rather than assigning numbers to each colors cant we just refer them via the color names ?